R/internal.R

```diff
@@ -88,11 +97,11 @@ uploadToS3 <- function(data, bucket, split_files) {
 
 #' @importFrom "aws.s3" "delete_object"
 #' @importFrom parallel mclapply
-deletePrefix <- function(prefix, bucket, split_files){
-  parallel::mclapply(1:split_files, function(i) {
-    s3Name = paste(prefix, ".", formatC(i, width = 4, format = "d", flag = "0"), ".psv.gz", sep="")
-    delete_object(s3Name, bucket)
-  })
+deletePrefix <- function(prefix, bucket, split_files) {
```

Comment:
This looks like it's only going to delete keys of a certain format within the prefix/folder, is that right? And we have to know the format?